app/assets/stylesheets/home.scss

```diff
@@ -1,9 +1,161 @@
-.container {
-    .webHeader{
-        display: inline;
-        #headerimg{
-             width: 50px;
-            height: 50px;
-        }
-    }
-}
+
+ div , h1 , a , table , span { padding: 0px; margin: 0px;}
+
+     a {

```

Comment:
Please keep good indentation, i think it's better that we use the id or class instead of the tap name, what do you think？